ipv-stub/src/endpoints/ipv-token.ts

```diff
@@ -66,7 +66,7 @@ function parseBody(body: string): Result<Partial<ValidatedParams>> {
 
   logger.info("Handling token request with parameters:");
   for (const [key, value] of Object.entries(validParameters)) {
-    const loggedValue = shouldObfuscate(key) ? obfuscate(value) : value;
+    const loggedValue = shouldObfuscate(key) ? truncate(value) : value;
```

Comment:
Nice, should we also change the `shouldObfuscate` function name? (I can see an argument for it either way so not too fussed)

Reply:
fwiw I have a slight preference for shouldTruncate but I'm also not fussed

Reply:
My thinking was we decide if we should obfuscate and then decide how, in this case just simply truncating.  If we get any more complex requirements we can change it then.